unlock-app/src/propTypes.js

```diff
@@ -66,6 +66,9 @@ export const element = PropTypes.oneOfType([PropTypes.func, PropTypes.element])
 
 export const delay = PropTypes.number
 
+export const items = PropTypes.array
```

Comment:
```suggestion
export const keyList = PropTypes.arrayOf(key)
```

Reply:
This is not correct. If the pagination component is reused somewhere else in the future, it could be an array of something else.

Reply:
We will update it then, go ahead with this for now 👍, as we don't have plans for pagination of anything else yet.